### PR TITLE
Use committer date instead of author date

### PR DESCRIPTION
**Pull request by halter73**
### Before

image

### After

image

The difference is in the AspNetCore-Tooling dependency. Both the before and after page correctly show the dependency is 7 commits behind, but the after page correctly calculates that the oldest unconsumed commit is 2 days old instead of 10.

[This commit](https://github.com/dotnet/aspnetcore-tooling/commits/af3a49ce3321410bba6b7e5c5fab7e94bc2b24b4) was tripping things up. It was authored 10 days ago, but was only merged into the aspnetcore-tooling repo yesterday.

@Pilchie @BrennanConroy

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "3.1.100"
```
Comment (Pilchie) [inline global.json:3]: Why do we need to pin this?

**Review by Pilchie — APPROVED** [review]
Looks good, I’ll merge and deploy tomorrow.

**Review by BrennanConroy — APPROVED** [review]

File: global.json (hunk 1 above)
Comment (halter73) [reply global.json:3]: My environment must have been in a bad state. I tried removing the global.json and everything now runs fine on my machine. Previously, when I tried to `dotnet run` this project, it was trying to use a prerelease version of the ASP.NET Core runtime which I don't have.

```
> dotnet run
It was not possible to find any compatible framework version
The framework 'Microsoft.AspNetCore.App', version '3.1.0-preview1.19508.20' was not found.
``` 

I don't know why it was choosing that runtime version. I didn't see any reference to it in the project, and I didn't have any other global.jsons further up the directory tree. Explicitly setting the SDK in the global.json did seem to fix the issue for me. @JunTaoLuo was there and can back me up. He was there.

Comment (halter73) [reply global.json:3]: Now that I think about it, I did only install version 3.1.100 of the SDK after making the global.json change. That might have been what really fixed things for me.

**Comment by Pilchie** [comment]
FYI that I'm getting timeouts trying to deploy to AppService Linux right now, but I'll keep trying.

**Comment by Pilchie** [comment]
Okay, this is deployed now.

**Pull request merged**